pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitRequestShippingTest.java

```diff
@@ -197,4 +210,21 @@ public void testPepperParticipantKitUpload() {
         });
     }
 
+    @Test
+    public void testNotHruidParticipantKitUpload() {
+        TransactionWrapper.inTransaction(conn -> {
+            String collaboratorParticipantId = "PROJ_" + notHruidId;
+            String collaboratorSampleId = collaboratorParticipantId + "_SALIVA";
+            //check when upload is made without hruid
```

Comment:
I do not understand this comment, since the following code does not appear to do an upload. 
It looks like a collaborator participant ID and BSP sample ID is synthesized.
What are we testing here?

Reply:
Sorry I didn't for it to be confusing and I totally see how it is now. I will change the comment